<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>baobiao</class>
 <widget class="QDialog" name="baobiao">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>637</width>
    <height>419</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="returnBtn">
       <property name="text">
        <string>返回</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>28</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>从</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateEdit" name="FromdateEdit">
       <property name="displayFormat">
        <string>yyyy-MM-dd</string>
       </property>
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>到</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateEdit" name="TodateEdit">
       <property name="displayFormat">
        <string>yyyy-MM-dd</string>
       </property>
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="searchBtn">
       <property name="text">
        <string>查询</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>          机型          </string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QTableWidget" name="JixingTab">
         <column>
          <property name="text">
           <string>日期</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>机型</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>安装数量</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>          usb口          </string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QTableWidget" name="UsbTab">
         <column>
          <property name="text">
           <string>日期</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>USB口</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>安装数量</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>总数：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="totalLabel">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>returnBtn</sender>
   <signal>clicked()</signal>
   <receiver>baobiao</receiver>
   <slot>OnReturn()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>57</x>
     <y>36</y>
    </hint>
    <hint type="destinationlabel">
     <x>71</x>
     <y>51</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>searchBtn</sender>
   <signal>clicked()</signal>
   <receiver>baobiao</receiver>
   <slot>OnSearchData()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>588</x>
     <y>41</y>
    </hint>
    <hint type="destinationlabel">
     <x>432</x>
     <y>64</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>OnReturn()</slot>
  <slot>OnSearchData()</slot>
 </slots>
</ui>
